checkov/terraform/checks/resource/aws/ImagebuilderImageRecipeEBSEncrypted.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, List, Any
+
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+
+
+class ImagebuilderImageRecipeEBSEncrypted(BaseResourceCheck):
+
+    def __init__(self):
+        name = "Ensure that Image Recipe EBS Disk are encrypted with CMK"
+        id = "CKV_AWS_200"
+        supported_resources = ["aws_imagebuilder_image_recipe"]
+        categories = [CheckCategories.ENCRYPTION]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
+        if conf.get('block_device_mapping'):
+            mappings = conf.get('block_device_mapping')
+            for mapping in mappings:
```

Comment:
I think you current code doesn't consider that the first block device is encrypted, but the second or any follow is not. Cann you add this test case and check, if you need to adjust your code.

Reply:
good catch. have it now